applications/openshift/scc/scc_limit_net_raw_capability/rule.yml

```diff
@@ -27,7 +27,26 @@ ocil_clause: 'NET_RAW does not exist or is configured in defaultAddCapabilities
 ocil: |-
     Inspect each SCC returned from running the following command:
     <pre>$ oc get scc</pre>
-    Review each SCC and determine that <tt>NET_RAW</tt> is either
+    Review each SCC and determine that either <tt>NET_RAW</tt> or <tt>ALL</tt> is either
     completely disabled as a list entry under <tt>requiredDropCapabilities</tt>,
-    or that <tt>NET_RAW</tt> is only enabled to a small set of
+    or that either <tt>NET_RAW</tt> or <tt>ALL</tt> is only enabled to a small set of
     containers and SCCs.
+
+{{%- if product == "ocp4" %}}
+warnings:
+    - general: |-
+        {{{ openshift_cluster_setting("/apis/security.openshift.io/v1/securitycontextconstraints") | indent(8) }}}
+{{%- endif %}}
+
+template:
+    name: yamlfile_value
+    vars:
+        ocp_data: "true"
+        filepath: /apis/security.openshift.io/v1/securitycontextconstraints
+        yamlpath: ".items[:]['requiredDropCapabilities']"
```

Comment:
I think we have the same issue as in PR #6250 with the fact that `requiredDropCapabilities` is an array...